src/http.c

```diff
@@ -156,7 +156,8 @@ int http_receive(struct tunnel *tunnel, char **response)
 
 			if (header_size) {
 				/* We saw the whole header, let's check if the
-				 * body is done as well */
+				 * body is done as well
+				 */
```

Comment:
Don't fully agree with this one, but why not.

Reply:
I have to say I don't agree either.
We can leave it out if you want to.

Reply:
In this case, let's leave it like it was!